### DISCUSSION TITLE
Akash Delegation Framework

### DISCUSSION THREAD

**Discussion by brewsterdrinkwater** [akash-network/community#379] [Akash Delegation Framework]
## New Delegations are currently **paused**. Please feel free to fill out the Delegation Applicant Form, and you will be added to the queue. 

Akash's security hinges on a global network of validators who not only maintain consensus but also actively contribute to developing the Akash Supercloud. These delegation, by Overclock Labs, offers AKT delegations as rewards to validators for exceptional contributions. To qualify, validators must satisfy certain baseline criteria, while additional efforts (detailed in Section 4) can greatly enhance the chances of receiving a delegation.

The Akash community welcomes any feedback on this framework. This is a living document that will evolve over time based on the network’s needs.

---

## Delegation Framework

Delegations will be earned, and maintained over time, by consistent, professional, and value-adding contributions to the Akash Network. While there may be some exceptions to the rule, Overclock Labs will govern delegations to Akash validators per the following principles and parameters.

### Minimum requirements:

- Must actively vote and participate in governance.
- Must be present for network upgrades (within 1 week of release).
- Must participate in one progress/check-in call per quarter.

### Additional contributions that will increase the likelihood of delegation:

- Build an open-source tool that will increase demand or supply on Akash.
- Maintain open-source repositories, databases, and more (and keep them running long-term).
- Contribute to the core Akash codebase.
- Participate consistently in Special Interest Group (SIGs), Working Groups (WGs) and User Group (UGs) for relevant areas of expertise.
- Technical integrations with other projects that drive workflows that contribute to network usage.
- Business development efforts that lead to more active supply or increased demand for network resources.
- Content creation that educates Akash users or builds awareness. This could include demos, walk-throughs, or general content highlighting various aspects of Akash.

#### **4.** Quarterly Delegation Reviews - Validators who receive delegations are required to check in quarterly to review contributions, discuss project updates, and review the delegation partnership.

---

If you have a desire to contribute on a short-term basis, please feel free to view the currently available bounties, various incentives (when applicable) or feel free to create a post on GitHub discussions if you would like to explore making a governance proposal with funding from the Akash community pool.

If you’re team is interested in receiving delegations, [please fill out this form](https://akashnet.typeform.com/to/kH2gpXOL). After submitting the form, you should receive an email from delegations@akash.network with next steps. If you haven’t received a reply within 10 days of submitting your application, please send a note in the #validators Discord channel with “@admin” cc-ed for further visibility, or feel free to send an email to delegations@akash.network.

Please feel free to ask any other specific questions in [#validators discord channel](https://discord.com/channels/747885925232672829/1111749461593575605/1172542701409947668).

**Comment by olisa001** [comment]
greetings, i filled out the form couple of weeks now, no response , please can someone take a look and respond ?
email used is wealth01@protonmail.com

**Reply by hewi333** [reply]
@olisa001 we have recieved your response. We currently have a back-log of responses. I will try to get you a formal response by end of week.

cc @brewsterdrinkwater
